### PR TITLE
44 create update round controller

### PR DESCRIPTION
**Pull request by MariusTulbure**
Fixes https://github.com/robert-damoc/game-stats/issues/44
Created/updated rounds_controller.

### CODE REVIEW

File: app/helpers/rounds_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module RoundsHelper
+end
```
Comment (robert-damoc) [inline app/helpers/rounds_helper.rb:2]: ?

File: app/controllers/games_controller.rb [hunk 2]
```diff
@@ -54,12 +54,12 @@ def set_game
   end
 
   def create_game_params
-    params.require(:game).permit(player_ids: [])
+    params.require(:game).permit(:rounds, player_ids: [])
```
Comment (robert-damoc) [inline app/controllers/games_controller.rb:57]: why?

File: app/controllers/games_controller.rb [hunk 3]
```diff
@@ -54,12 +54,12 @@ def set_game
...
   end
 
   def update_game_params
     if @game.state_created?
-      params.require(:game).permit(:state, player_ids: [])
+      params.require(:game).permit(:state, :rounds, player_ids: [])
```
Comment (robert-damoc) [inline app/controllers/games_controller.rb:62]: why?

File: app/models/player.rb [hunk 4]
```diff
@@ -3,6 +3,7 @@ class Player < ApplicationRecord
 
   has_many :game_players, -> { order(position: :asc) }, dependent: :destroy, inverse_of: :player
   has_many :games, through: :game_players
+  has_many :rounds, dependent: :destroy
```
Comment (robert-damoc) [inline app/models/player.rb:6]: Is the player directly associated with a round?

**Review by robert-damoc — COMMENTED** [review]
Why do we have UI changes on this PR?

File: app/controllers/games_controller.rb (hunk 2 above)
Comment (MariusTulbure) [reply app/controllers/games_controller.rb:57]: deleted

File: app/controllers/games_controller.rb (hunk 3 above)
Comment (MariusTulbure) [reply app/controllers/games_controller.rb:62]: deleted

File: app/models/player.rb (hunk 4 above)
Comment (MariusTulbure) [reply app/models/player.rb]: deleted

**Review by robert-damoc — APPROVED** [review]

**Pull request merged**